youtbestyle2.css

```diff
@@ -18,7 +18,7 @@
 }
 #bars{
     background-color: blue;
-    /* font-size: larger; */
```

Comment:
It is always suggested to remove the comments from the code base, this will make the code look clean.